plugins/hydra_colorlog/pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[tool.towncrier]
```

Comment:
I think you should keep:
feature, bugfix, config and docs.

Reply:
We have had some instances of other types in Ax plugin. For example:

1. api_change -> https://github.com/facebookresearch/hydra/issues/913
2. maintenance -> https://github.com/facebookresearch/hydra/pull/782/files

Reply:
@omry ^^

Reply:
Doesnt seem obvious. Will look around more

Reply:
Does not seem straightforward. Should we merge this, as is?

Reply:
Yes